### ISSUE TITLE
move the buy-all-upgrades button

### ISSUE THREAD

**Issue by erosson** [swarmsim/swarm#471]
the no-unit-selected screen seems like a good choice. it's awfully inconvenient to click it several times in a row when it's in a menu, and there's at least one report of someone clicking it accidentally when looking for the options screen.

moving it outside of a menu would make it easier to autoclick, unfortunately. probably okay with that; cheaters gonna cheat. could always make the button move around the screen once in a while if this gets to be a problem.

**Comment by Shoelace** [comment]
i hit an issue with this the other day..  i wanted to use buy all upgrades.. but it kept empowering my territory and  i ended up with less stuff.

so i think 2 improvements coudl be good.
1: buy-all by tab..
or 2: only buy upgrades i have notifications turned on for.

**Comment by erosson** [comment]
> 2: only buy upgrades i have notifications turned on for.

I'd thought it was already doing that. If it's really buying *all*, not just upgrades with notifications, that's a bug.

**Comment by erosson** [comment]
related to #489

**Comment by Shoelace** [comment]
no.. my fault.. it is doing that.. i forgot the when i ascended i had reset all the notifications.. :( 

i've tried again and buy-all-upgrade is not buying unnotifed stuff ..  sorry.

**Comment by erosson** [comment]
No worries.

**Comment by 1000i100** [comment]
>  1: buy-all by tab..

I aprove this idea !
